OPAL/tac/src/main/scala/org/opalj/tac/fpcf/analyses/cg/AndroidICCAnalysis.scala

```diff
@@ -0,0 +1,1413 @@
+/* BSD 2-Clause License - see OPAL/LICENSE for details. */
+package org.opalj
+package tac
+package fpcf
+package analyses
+package cg
+
+import org.opalj.br.{ClassFile, Method, ObjectType, ObjectVariableInfo}
+import org.opalj.br.analyses.{DeclaredMethodsKey, ProjectInformationKeys, SomeProject}
+import org.opalj.br.fpcf.properties.SimpleContextsKey
+import org.opalj.br.instructions.{INVOKESPECIAL, INVOKESTATIC, INVOKEVIRTUAL, Instruction}
+import org.opalj.collection.immutable.UIDSet
+import org.opalj.fpcf.{OrderedProperty, PartialResult, ProperPropertyComputationResult, Property, PropertyBounds, PropertyKey, PropertyMetaInformation, PropertyStore, Results}
+import org.opalj.value.ValueInformation
+
+import scala.collection.mutable
+import scala.collection.mutable.ListBuffer
+import org.opalj.br.fpcf.{BasicFPCFEagerAnalysisScheduler, FPCFAnalysis}
+import org.opalj.tac.fpcf.properties.cg.{Callees, Callers}
+
+import scala.io.Source
+import scala.xml.{Elem, XML}
+
+sealed trait AndroidICCPropertyMetaInformation extends PropertyMetaInformation {
```

Comment:
What is this property? It seems to be unused (and useless anyway, since you're just computing call graph data).